async-clipboard/demo.js

```diff
@@ -0,0 +1,55 @@
+const copy = document.querySelector('#copy');
+const paste = document.querySelector('#paste');
+const out = document.querySelector('#out');
+
+/** Write contents of the textarea to the clipboard when clicking "Copy" */
+copy.onclick = () => {
```

Comment:
I'd recommend something like `document.querySelector('#copyButton').addEventListener('click', () => {` so that you don't have the `const` declaration.
Same for `paste`.

Reply:
Done.